src/index.mjs

```diff
@@ -398,3 +400,11 @@ export function generateLabelRange(min, max, numLabels) {
 
   return labels;
 }
+
+export function getWidth(fontSize, dataPoints) {
+  const characterHeight = Number(fontSize) // This height is relative to viewbox and not in px
+  if(isNaN(characterHeight)) throw new Error('Invalid fontSize')
+  const characterWidth = characterHeight / 2 // For fonts such as Helvetica and Arial the ratio of height to width is ~2
+  const maxWidth = characterWidth * (String(Math.max(...dataPoints)).length)
+  return (maxWidth + 2) // Add padding
```

Comment:
Declare magic constant. Also: Parenthesis not needed?

Reply:
This is a general problem. Can we put `.prettierrc` in the repo?

Reply:
makes sense. u can create a new issue that can get paid separately and we leave the PR as is for now.